infra/src/main/java/com/redhat/service/bridge/infra/models/actions/BaseAction.java

```diff
@@ -13,10 +13,6 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
I think you have to add here `@JsonIgnoreProperties(ignoreUnknown = true)` otherwise the actions present in the demo database will crash the deserialization. Can you double check? You might
1) start the manager from `main` branch
2) create a processor/action with a name
3) start the manager from this PR 
4) try to retrieve the processor created at step 2

Reply:
@r00ta Surprisingly it did not throw an exception; however I think it will be good practice to include the annotation. 

I will add it in a new commit.